modules/src/Components/DisplayTodo.tsx

```diff
@@ -0,0 +1,72 @@
+// Components/DisplayTodo.tsx
+import { useEffect, useState } from "react";
+import { useFetch } from "../Hooks/UseFetch";
+import { Todo } from "./Home";
+import Task from "./Task";
+
+import { API_ENDPOINT } from "../constants";
+
+const DisplayTodo = () => {
+  const [isCompleted, setisCompleted] = useState<boolean>(false);
+  const { loading, error, data } = useFetch();
+  const [tasks, setTasks] = useState<Todo[]>([]);
+
+  useEffect(() => {
+    setTasks(data);
+  }, [data]);
+
+  // ---------------------------- Delete Task ----------------------------
+  const deleteTask = async(id: string) => {
+    const filteredTasks = tasks.filter((task) => task.id !== id);
+    setTasks(filteredTasks);
+
```

Comment:
If delete is happening in through API, data has to refecth once delete req is success. It is not recommended to update the local state when api integration is there.